### DISCUSSION TITLE
Docs for Sticky are broken

### DISCUSSION THREAD

**Discussion by jkinley** [TanStack/virtual#348] [General]
https://codesandbox.io/s/github/tanstack/virtual/tree/beta/examples/react/sticky?from-embed

<img width="1628" alt="image" src="https://user-images.githubusercontent.com/19417886/181268270-6fea689b-6812-4002-be4a-9ace72f5bfdf.png">

**Comment by ardeora** [comment]
Thanks for pointing that out. I have a PR in place to fix this https://github.com/TanStack/virtual/pull/349 Meanwhile, you can see the codesandbox linked below to see the Sticky Example

https://codesandbox.io/s/youthful-estrela-vggt56?file=/src/main.tsx

**Reply by jkinley** [reply]
@ardeora No problem. Thanks for all your hard work on this.
